pkg/monitoring/domainstats/prometheus/prometheus_test.go

```diff
@@ -35,6 +35,10 @@ import (
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/stats"
 )
 
+var emptyVmFSStats = k6tv1.VirtualMachineInstanceFileSystemList{
+	Items: []k6tv1.VirtualMachineInstanceFileSystem{},
+}
+
```

Comment:
Having global mutabale variables for tests is a bad practice. I think you should choose of the two options:
1. Declare the variable globally but initialize it in a `BeforeEach` clause
2. Replace the variable with a function that returns an empty VMFSStats

Option 2 is safer and better imho, it isolates tests from one another and more readable

Reply:
updated